plugins/semanticcache/docker-compose.yml

```
@@ -0,0 +1,264 @@
+    container_name: redis_cluster_init
+    depends_on:
+      redis-cluster-node-1:
+        condition: service_healthy
+      redis-cluster-node-2:
+        condition: service_healthy
+      redis-cluster-node-3:
+        condition: service_healthy
+    command: |
+      sh -c '
+        echo "Starting Redis cluster initialization and monitoring..."
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)